extensions/2.0/Khronos/KHR_lights_punctual/schema/light.schema.json

```diff
@@ -51,8 +52,7 @@
         "range": {
             "type": "number",
             "description": "A distance cutoff at which the light's intensity may be considered to have reached zero.",
-            "minimum": 0,
-            "exclusiveMinimum": true
+            "exclusiveMinimum": 0
```

Comment:
Nitpick: Since this value is not restricted to integers, I would prefer `0.0` instead of just `0`, to make it clear to readers. This would also be consistent with [other values](https://github.com/KhronosGroup/glTF/pull/2380/files#diff-fe74ec687a590b4d07790b3deed9e0c93dd941cc960506fdfa67602fdb4de120R23).

Same for `light.spot.schema.json`.